src/config.rs

```diff
@@ -194,7 +194,7 @@ impl Config {
 
         f(dest, Arc::new(self.clone())).await;
 
-        self.destinations.remove(&logplex_token.to_owned());
```

Comment:
what, is rust stable doing some transparent ownership / casting magic now?

Reply:
it always did that, so the call to `.to_owned` wasn't necessary from the start. 

Now clippy catches this.